pkg/scheduler/internal/cache/debugger/comparer.go

```diff
@@ -57,11 +57,11 @@ func (c *CacheComparer) Compare() error {
 	pendingPods := c.PodQueue.PendingPods()
 
 	if missed, redundant := c.CompareNodes(nodes, dump.Nodes); len(missed)+len(redundant) != 0 {
-		klog.Warningf("cache mismatch: missed nodes: %s; redundant nodes: %s", missed, redundant)
+	    klog.InfoS("Cache mismatch", "missedNodes", missed, "redundantNodes", redundant)
```

Comment:
do not change format. BTW, not sure about `missedNodes` such columns can provide real help. Leave it other reviewers.